Modules/System/Cryptography Management/src/CryptographyManagementImpl.Codeunit.al

```diff
@@ -470,6 +470,11 @@ codeunit 1279 "Cryptography Management Impl."
         DataOutStream.WriteText(InputString);
         exit(VerifyData(DataInStream, XmlString, HashAlgorithm, SignatureInStream));
     end;
+    
+    procedure VerifyData(InputString: Text; SignatureKey: Codeunit "Signature Key"; HashAlgorithm: Enum "Hash Algorithm"; SignatureInStream: InStream)
```

Comment:
Quick and minor change, as this doesn't compile.

Needs to return boolean and enclose the VerifyData call in exit()